src/data/faq.json

```diff
@@ -555,7 +555,7 @@
                         "anchor": "check_in_ prevent_false_warnings",
                         "active": true,
                         "textblock": [
-                            "<ul><li>Check in only for events or places you visit. Do not scan Event QR codes distributed through digital media or websites</li><li>Check-in to see if the title and location of the event (shown in the app after scanning) match the event you attended</li><li>The more accurately you specify the time of your attendance, the more accurately you can be warned or warn other participants</li><li>Already when checking in, specify as accurately as possible when you want to be checked out automatically. If your stay is shortened or extended, you can also manually correct the time after the automatic check-out has already taken place</li><li>Don't share photographed Event QR codes via social media or other channels</li></ul>",
+                            "<ul><li>Check-In only for events or places you visit. Do not scan Event QR codes distributed through digital media or websites</li><li>Check-In to see if the title and location of the event (shown in the app after scanning) match the event you attended</li><li>The more accurately you specify the time of your attendance, the more accurately you can be warned or warn other participants</li><li>Already when checking in, specify as accurately as possible when you want to be checked out automatically. If your stay is shortened or extended, you can also manually correct the time after the automatic check-out has already taken place</li><li>Don't share photographed Event QR codes via social media or other channels</li></ul>",

```

Comment:
In this case "Check in" is a verb, so there should not be a hyphen here.
See https://dictionary.cambridge.org/dictionary/english/check-in

It was originally inconsistent so both instances need to be written as "Check in" because they are both verbs.

Reply:
Thank you! Done in https://github.com/corona-warn-app/cwa-website/pull/1481/commits/a88aa45202575f309bcad2f2e0767b2f3ecd0727.